star_ratings/models.py

```diff
@@ -113,6 +113,12 @@ def has_rated(self, instance, user=None):
         rating = self.for_instance_by_user(instance, user=user)
         return rating is not None
 
+    def bulk_create(self, objs, batch_size=None):
+        objs = super(UserRatingManager, self).bulk_create(objs, batch_size=None)
+        for obj in objs:
```

Comment:
I'm fine leaving this as is for now but maybe we should look at only looping over unique ratings as this will call `calculate` on the same rating twice